sebs/azure/azure.py

```diff
@@ -513,8 +575,40 @@ def enforce_cold_start(self, functions: List[Function], code_package: Benchmark)
     """
 
     def create_trigger(self, function: Function, trigger_type: Trigger.TriggerType) -> Trigger:
-        raise NotImplementedError()
+        from sebs.azure.triggers import QueueTrigger, StorageTrigger
 
+        azure_function = cast(AzureFunction, function)
+        resource_group = self.config.resources.resource_group(self.cli_instance)
+        storage_account = azure_function.function_storage.account_name
+
+        ret = self.cli_instance.execute(
+            ('az storage account show --resource-group {} --name {} --query id')
+            .format(resource_group, storage_account)
+        )
+        self.cli_instance.execute(
+            ('az role assignment create --assignee "{}" \
+              --role "Storage {} Data Contributor" \
+              --scope {}')
+            .format(
+                os.environ["AZURE_USER_PRINCIPAL_NAME"],
```

Comment:
I think I understand where this data might be coming from. Is this in format similar to `${name}_${email}.com#EXT#@${name}${email}.onmicrosoft.com`?

If yes, we can get this by running `az ad user list`. I found this here: https://github.com/MicrosoftDocs/azure-docs/blob/main/articles/role-based-access-control/role-assignments-cli.md